test/interface-tests/src/dag/import.js

```diff
@@ -119,31 +119,46 @@ export function testImport (factory, options) {
       }
     })
 
-    it('should import car with roots but no blocks', async () => {
+    it('should import car with roots but no blocks', async function () {
       const input = loadFixture('test/interface-tests/fixtures/car/combined_naked_roots_genesis_and_128.car')
+
       const reader = await CarReader.fromBytes(input)
       const cids = await reader.getRoots()
 
       expect(cids).to.have.lengthOf(2)
 
+      let result = await all(ipfs.dag.import((async function * () { yield input }())))
+
+      expect(result).to.have.lengthOf(2)
       // naked roots car does not contain blocks
-      const result1 = await all(ipfs.dag.import(async function * () { yield input }()))
-      expect(result1).to.deep.include({ root: { cid: cids[0], pinErrorMsg: 'blockstore: block not found' } })
-      expect(result1).to.deep.include({ root: { cid: cids[1], pinErrorMsg: 'blockstore: block not found' } })
+      expect(result[0].root.cid.toString()).to.equal(cids[0].toString())
+      expect(result[0].root.pinErrorMsg).to.be.a('string')
+      expect(result[0].root.pinErrorMsg).to.not.be.empty(`result[0].root.pinErrorMsg should not be empty but contains ${result[0].root.pinErrorMsg}`)
+      expect(result[1].root.cid.toString()).to.equal(cids[1].toString())
+      expect(result[1].root.pinErrorMsg).to.be.a('string')
+      expect(result[1].root.pinErrorMsg).to.not.be.empty(`result[1].root.pinErrorMsg should not be empty but contains ${result[1].root.pinErrorMsg}`)
 
       await drain(ipfs.dag.import(async function * () { yield loadFixture('test/interface-tests/fixtures/car/lotus_devnet_genesis_shuffled_nulroot.car') }()))
+      result = await all(ipfs.dag.import((async function * () { yield input }())))
 
-      // have some of the blocks now, should be able to pin one root
-      const result2 = await all(ipfs.dag.import(async function * () { yield input }()))
-      expect(result2).to.deep.include({ root: { cid: cids[0], pinErrorMsg: '' } })
-      expect(result2).to.deep.include({ root: { cid: cids[1], pinErrorMsg: 'blockstore: block not found' } })
+      expect(result).to.have.lengthOf(2)
+      expect(result[0].root.cid.toString()).to.equal(cids[0].toString())
+      expect(result[0].root.pinErrorMsg).to.be.a('string')
+      expect(result[0].root.pinErrorMsg).to.be.empty()
+      expect(result[1].root.cid.toString()).to.equal(cids[1].toString())
+      expect(result[1].root.pinErrorMsg).to.be.a('string')
+      expect(result[1].root.pinErrorMsg).to.not.be.empty(`result[1].root.pinErrorMsg should not be empty but contains ${result[1].root.pinErrorMsg}`)
```

Comment:
yea, good catch

1. yep, message should probably say something like "should not be empty" instead. 
1. Because that error is no longer the same, and hardcoding tests to specific error strings (that will change. hence this change) is not a great practice. ideally, we would have an error code we could check that doesn't change as we adapt the language in the message, but we don't. We could switch to looking for "find/found" and "not" or something, but at that point we should probably hardcode the new error message. 

I'm willing to hardcode the new error message if that's what we want

Reply:
fixed and re-enabled, though one part is still failing partially. see https://github.com/ipfs/js-kubo-rpc-client/issues/94